basketball_reference_web_scraper/parsers/players_season_advanced.py

```diff
@@ -0,0 +1,60 @@
+from lxml import html
+
+from basketball_reference_web_scraper.data import TEAM_ABBREVIATIONS_TO_TEAM, POSITION_ABBREVIATIONS_TO_POSITION
+from basketball_reference_web_scraper.utilities import str_to_int
+
+
+def parse_player_season_advanced(row):
+    return {
+        "slug": str(row[1].get("data-append-csv")),
+        "name": str(row[1].text_content()),
+        "positions": parse_positions(row[2].text_content()),
+        "age": str_to_int(row[3].text_content(), default=None),
+        "team": TEAM_ABBREVIATIONS_TO_TEAM.get(row[4].text_content()),
+        "games_played": str_to_int(row[5].text_content()),
+        "minutes_played": str_to_int(row[6].text_content()),
+        "player_efficiency_rating": row[7].text_content(),
```

Comment:
Should these numbers with decimals be `str_to_float`ed?

I know it's annoying to convert these values (and probably not very Python-ic) but I think it's nicer for client users to have values in types that make sense vs. having to convert it on their end.